website/docs/docs/dbt-cloud/dbt-cloud-enterprise/sso-overview.md

```diff
@@ -16,7 +16,9 @@ dbt Cloud supports JIT (Just-in-Time) provisioning and IdP-initiated login. You
 
 The diagram below explains the basic process by which users are provisioned in dbt Cloud upon logging in with SSO.
 
-![SSO-diagram](https://user-images.githubusercontent.com/46451573/84216940-e0b60e80-aa98-11ea-91a2-4bffbfffe69e.png)
+<Lightbox src="/img/sso_overview.png" title="SSO-diagram" />
+
+![SSO-diagram](static/img/sso_overview.png)
```

Comment:
i think we do not want this image here, right?

Reply:
@drewbanin This is actually a revised version of the SSO diagram, without Auth0 in the mix. :)

Reply:
yeah - it's a great diagram! But we can just keep the `Lightbox` and remove the `![SSO-diagram]` image embed which is not rendering on the page.

<img width="753" alt="Screen Shot 2020-10-20 at 8 34 10 AM" src="https://user-images.githubusercontent.com/1541139/96586544-238dbb80-12af-11eb-9e5a-501ba7e63fbb.png">

I don't think we need/want both of these image embeds here.....

Reply:
Ahhhh I see what you mean. That was an accident 😓 . Fixing that now :)